File: Android/IOTDemo_jwt/app/src/main/java/com/mitlab/iotdemo_jwt/model/WorkerTopic.java
package com.mitlab.iotdemo_jwt.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WorkerTopic {
    @Expose
    @SerializedName("id")
    int id;

    @Expose
    @SerializedName("worker_type_id")
    long worker_type_id;

    @Expose
    @SerializedName("topic_name")
    String topic_name;

    @Expose
    @SerializedName("topic_path")
    String topic_path;

    @Expose
    @SerializedName("topic_description")
    String topic_description;
}

File: Web/IOTDemo_jwt/app/WorkerTopic.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class WorkerTopic extends Model
{
        /**
     * @var string
     */
    protected $table = 'worker_topics';
    /**
     * @var array
     */
    protected $guarded = [];
    protected $fillable = [
        'worker_type_id',
        'topic_name',
        'topic_path',
        'topic_description'
    ];
    protected $hidden = [
        'laravel_through_key', 'created_at', 'updated_at'
    ];
    public function workers()
    {
        return $this->hasMany('App\Worker', 'worker_type_id', 'worker_type_id');
    }
}
